### PR TITLE
Inactive branches

### PR DESCRIPTION
**Pull request by philkra**
- what effect does an inactive branch have in my workspace
- describe the duration and events for a branch to become inactive

### CODE REVIEW

**Comment by philkra** [comment]
Preview: https://xata-docs-pr-342.vercel.app/docs/inactive-branches

File: 100-Appendix/035-inactivity.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+keywords: ['branch', 'search']
+description: Actions take after branch inactivity
+slug: inactive-branches
+published: true
+---
+
+If no traffic is detected for a branch over a 30-day period, either through the REST API or direct [Postgres access](/docs/postgres), Xata will flag the branch as inactive and take appropriate action.
+As a result, all search-related operations will be turned off, including [full-text search](/docs/sdk/search), [aggregations](/docs/sdk/aggregate), [vector search](/docs/sdk/vector-search), and [Ask AI](/docs/sdk/ask).
+However, other features from the REST API, [file attachments](/docs/sdk/file-attachments) and direct [Postgres access](/docs/postgres) will remain available.
+To re-enable the search engine, navigate to the branch management section in the corresponding workspace and turn the search engine on for the desired branch.
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:13]:
```suggestion
To re-enable the search engine, navigate to the database settings section in the corresponding workspace and turn the search engine on for the desired branch.
```

Settings live at the db level, they just show at any branch route.

File: 100-Appendix/035-inactivity.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Branch Inactivity
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:2]:
```suggestion
title: Branch inactivity
```

File: 100-Appendix/035-inactivity.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+navTitle: Branch Inactivity
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:3]:
```suggestion
navTitle: Branch inactivity
```

File: 100-Appendix/035-inactivity.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+keywords: ['branch', 'search']
+description: Actions take after branch inactivity
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:5]:
```suggestion
description: Branches are set to inactive after a period of time if not in use.
```

File: 100-Appendix/035-inactivity.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+slug: inactive-branches
+published: true
+---
+
+If no traffic is detected for a branch over a 30-day period, either through the REST API or direct [Postgres access](/docs/postgres), Xata will flag the branch as inactive and take appropriate action.
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:10]:
```suggestion
If no traffic is detected for a branch for a full 30-day period, either through the REST API or direct [Postgres access](/docs/postgres), Xata will flag the branch as inactive and disable features.
```

File: 100-Appendix/035-inactivity.mdx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+As a result, all search-related operations will be turned off, including [full-text search](/docs/sdk/search), [aggregations](/docs/sdk/aggregate), [vector search](/docs/sdk/vector-search), and [Ask AI](/docs/sdk/ask).
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:11]:
```suggestion
Once set to inactive, the search engine will be disabled. This includes [full-text search](/docs/sdk/search), [aggregations](/docs/sdk/aggregate), [vector search](/docs/sdk/vector-search), and [Ask AI](/docs/sdk/ask).
```

File: 100-Appendix/035-inactivity.mdx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+However, other features from the REST API, [file attachments](/docs/sdk/file-attachments) and direct [Postgres access](/docs/postgres) will remain available.
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:12]:
```suggestion
This only effects search. All other portions of the REST API, [file attachments](/docs/sdk/file-attachments) and direct [Postgres access](/docs/postgres) will remain available.
```

File: 100-Appendix/035-inactivity.mdx [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+To re-enable the search engine, navigate to the branch management section in the corresponding workspace and turn the search engine on for the desired branch.
+
+This action only applies to workspaces on the [Free plan](/docs/concepts/pricing#free-plan).
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:15]:
```suggestion
Features will only be disabled on inactive branches in workspaces on our [Free plan](/docs/concepts/pricing#free-plan).
```

File: 100-Appendix/035-inactivity.mdx [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+To avoid this action, please upgrade the workspace to a [Pro plan](/docs/concepts/pricing#pro-plan).
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:16]:
```suggestion
```

This this is an an obvious statement that can be removed for clarity.

File: 100-Appendix/035-inactivity.mdx [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+Inactive branches will still count towards the [limits](/pricing) imposed by the [Free plan](/docs/concepts/pricing#free-plan).
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:17]:
```suggestion
Branches set to inactive will still count towards the [limits](/pricing) imposed by the [Free plan](/docs/concepts/pricing#free-plan).
```

File: 100-Appendix/035-inactivity.mdx [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+Xata will send you a notification via email ten days before taking any action, if no activity continues for a 30 days window in total, Xata will take action on the branch.
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:19]:
```suggestion
Xata sends a notification by email ten days before setting a branch to inactive. If the branch continues to not be used, Xata will mark it as inactive and you'll receive another email notification.
```

File: 100-Appendix/035-inactivity.mdx [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+Afterwards, you'll receive another email once the branch has been marked as inactive, after the 30-day period.
```
Comment (snide) [inline 100-Appendix/035-inactivity.mdx:20]:
```suggestion
```

Combined with above sentence

**Review by snide — COMMENTED** [review]
Small clarity cleanup. Thanks for writing thisup.

File: 100-Appendix/035-inactivity.mdx [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+Once set to inactive, the search engine will be disabled. This includes [full-text search](/docs/sdk/search), [aggregations](/docs/sdk/aggregate), [vector search](/docs/sdk/vector-search), and [Ask AI](/docs/sdk/ask).
+This only effects search. All other portions of the REST API, [file attachments](/docs/sdk/file-attachments) and direct [Postgres access](/docs/postgres) will remain available.
+To re-enable the search engine, navigate to the database settings section in the corresponding workspace and turn the search engine on for the desired branch.
+
+Features will only be disabled on inactive branches in workspaces on our [Free plan](/docs/concepts/pricing#free-plan).
+Branches set to inactive will still count towards the [limits](/pricing) imposed by the [Free plan](/docs/concepts/pricing#free-plan).
+
+Xata sends a notification by email ten days before setting a branch to inactive. If the branch continues to not be used, Xata will mark it as inactive and you'll receive another email notification.
```
Comment (tsg) [inline 100-Appendix/035-inactivity.mdx:18]: Can we use "a few days before" instead of ten? Because I'll give people a shorter heads-up this time.

**Review by tsg — APPROVED** [review]

File: 100-Appendix/035-inactivity.mdx (hunk 13 above)
Comment (philkra) [inline 100-Appendix/035-inactivity.mdx:18]:
```suggestion
Xata sends a notification by email a few days before setting a branch to inactive. If the branch continues to not be used, Xata will mark it as inactive and you'll receive another email notification.
```

Comment (philkra) [reply 100-Appendix/035-inactivity.mdx]: adjusted with https://github.com/xataio/mdx-docs/pull/342/commits/b179c7b91798a1bc6e5f1d56601395db9a948f8c -- thanks

**Pull request merged**